### ISSUE TITLE
problems of BLEU and SELF-BLEU on test data of COCO in Table 5 in the paper

### ISSUE THREAD

**Issue by anonymous1100** [pclucas14/GansFallingShort#7]
Hi,
I found a problem with the training data in Table 5 in your paper. COCO's selfbleu5 is bigger than selfbleu4. Here is the BLEU and SELF-BLEU I tested on test data of COCO.
bleu2=0.744 bleu3=0.526 bleu4=0.344 bleu5=0.222
self-bleu2=0.895 self-bleu3=0.751 self-bleu4=0.581 self-bleu5=0.422

**Comment by optimass** [comment]
Thanks for pointing this out! There is indeed a typo.
May we ask what command you ran?

**Comment by cp-pc** [comment]
I took your COCO data set and tested it on our code. The metric code is the same as yours.
